### PR TITLE
[13.0][MIG] website_media_size

### PR DESCRIPTION
**Pull request by tarteo**

### CODE REVIEW

File: website_media_size/static/src/js/website_media_size.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* Copyright 2018 Onestein
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('website_media_size', function (require) {
+    "use strict";
+
+    var MediaModules = require('wysiwyg.widgets.media');
```
Comment (Tardo) [inline website_media_size/static/src/js/website_media_size.js:7]:
```suggestion
    const MediaModules = require('wysiwyg.widgets.media');
```

File: website_media_size/static/src/js/website_media_size.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    var ImageWidget = MediaModules.ImageWidget;
```
Comment (Tardo) [inline website_media_size/static/src/js/website_media_size.js:8]:
```suggestion
    const ImageWidget = MediaModules.ImageWidget;
```

File: website_media_size/static/src/xml/website_media_size.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<template>
+    <t t-extend="wysiwyg.widgets.image.existing.attachment">
+        <t t-jquery=".o_existing_attachment_cell > div" t-operation="append">
+            <t t-if="attachment.type != 'url'">
+                <t t-set="file_size" t-value="Math.ceil(attachment.file_size / 1024)"/>
+                <span t-attf-class="o_existing_attachment_size #{file_size > 999 and 'o_existing_attachment_size_danger' or file_size > 99 and 'o_existing_attachment_size_warning' or ''}"><t t-esc="file_size"/> kB</span>
```
Comment (Tardo) [inline website_media_size/static/src/xml/website_media_size.xml:10]: I don't like these hard-coded values (999 and 99)... and units are not correct, for me better use "KiB".

File: website_media_size/templates/assets.xml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <template id="assets_editor" inherit_id="website.assets_editor">
+        <xpath expr=".">
+            <script type="text/javascript"
+                    src="/website_media_size/static/src/js/website_media_size.js"/>
+
+            <link href="/website_media_size/static/src/scss/website_media_size.scss"
+                  type="text/scss" rel="stylesheet"/>
```
Comment (Tardo) [inline website_media_size/templates/assets.xml:12]:
```suggestion
                  type="text/css" rel="stylesheet"/>
```

**Review by Tardo — CHANGES_REQUESTED** [review]
Thanks for the work

**Review by claudiagn — COMMENTED** [review]
@tarteo can you provide a module description so I can review please? thx 👍

**Pull request closed without merging**